server/cluster/cluster.go

```diff
@@ -1176,6 +1181,34 @@ func (c *RaftCluster) UpStore(storeID uint64) error {
 	return c.putStoreLocked(newStore)
 }
 
+// ReadyToServe change store's node state to Serving.
+func (c *RaftCluster) ReadyToServe(store *core.StoreInfo) error {
+	c.Lock()
```

Comment:
What are we try to protect before L1208?

Reply:
> What are we try to protect before L1208?

Maybe to protect the store from a state change

Reply:
However, I think it does not work.

Reply:
I modified the function implementation to keep consistent with others, PTAL.